rebase_v1.13.1_action_plan.md

```diff
@@ -0,0 +1,81 @@
+# CoreDNS v1.13.1 Rebase Action Plan
+
+_Status legend:_ ✅ complete · 🔄 in progress · ⬜️ pending · 🚫 blocked
+
+## Milestone Summary
+
+| Status | Task | Notes |
+| --- | --- | --- |
+| ✅ | Produce carry audit (`audit_upstream_report.md`) | Completed; classifies all `UPSTREAM`/`<carry>` commits |
+| ✅ | Draft rebase analysis (`rebase_v1.13.1_report.md`) | Completed with appendix and consolidation plan |
+| ✅ | Prototype build with rebased `ocp_dnsnameresolver` | Prototype branch `prototype/v1.13.1-with-ocp`; Linux tests passing |
+| ✅ | Publish rebased `coredns-ocp-dnsnameresolver` module | Upstream merged; CoreDNS `go.mod` points to `v0.0.0-20251118…` |
+| ✅ | Execute full rebase of `origin/main` onto upstream `v1.13.1` | Merge commit staged; downstream adjustments applied |
+| ✅ | Cut downstream release notes draft | See `notes/coredns-1.13.1-openshift.md` |
+| 🔄 | Final CI/regression sweep & hand-off | Waiting on prow rehearsals; schedule rebase review meeting |
+| ⬜️ | Coordinate stakeholder review meeting | Prep Google Meet to walk through rebase status |
+
+## Detailed Checklist
+
+### 1. Tooling & Environment Prep
+- ✅ Confirm builds succeed with Go 1.24 using `GOTOOLCHAIN=local`.
+- ⬜️ Ensure CI/builder configs set `GOTOOLCHAIN=local` (or equivalent) so the same toolchain is used everywhere.
+
+### 2. Carry Commit Consolidation
```

Comment:
Knowing why (and which) we squashed carry patches is the only thing which may have a reason to be carried.